opennlp-tools/src/main/java/opennlp/tools/tokenize/lang/Factory.java

```diff
@@ -25,24 +25,45 @@
 
 public class Factory {
 
-  public static final String DEFAULT_ALPHANUMERIC = "^[A-Za-z0-9]+$";
+  public static final Pattern DEFAULT_ALPHANUMERIC = Pattern.compile("^[A-Za-z0-9]+$");
```

Comment:
Is this used elsewhere (aside from inside tests)? Otherwise, we could just make it private to be consistent with the other constant patterns?

Reply:
Others might reference this. That's why I left its visibility untouched.

Even though the type changed to Pattern, the original forma can easily be accessed via `.pattern()` which returns it as a String.

Reply:
I am fine with the reasoning for keeping it public atm, but would prefer to make it private in the long run (maybe via the @Deprecated approach as with `alphaNumeric` before). 🙃